code/modules/shuttle/marine_dropship.dm

```diff
@@ -620,7 +626,7 @@
 	crashing_dropship.crashing = TRUE
 	crashing_dropship.unlock_all()
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_DROPSHIP_HIJACKED)
-	priority_announce("Unscheduled dropship departure detected from operational area. Hijack likely. Shutting down autopilot.", "Dropship Alert", sound = 'sound/AI/hijack.ogg')
+	priority_announce("Unscheduled dropship departure detected from operational area. Hijack likely.", "Dropship Alert")
```

Comment:
Revert this

Reply:
We already say that the autopilot was shut down in a prior annoncment (when shrike call down alamo). Would better to change the sound, maybe just removing the last sentence

Reply:
Added back sound